public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -190,7 +190,10 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
   }
 
   public startUrlSync() {
-    if (!this.state.meta.isEmbedded) {
+    if (
+      !this.state.meta.isEmbedded &&
+      !(config.publicDashboardAccessToken && this.state.controls?.state.hideTimeControls)
```

Comment:
I wonder if it would be better to add this condition in `loadDashboard` DashboardScenePageStateManager method

Reply:
Do you mean, add `this.state.meta.isPublicDashboard`? I'd do this instead.

Reply:
no, I mean. `startUrlSync()` is called by `loadDashboard()` and also in `useSaveDashboard`. I could add this 
`!(config.publicDashboardAccessToken` && this.state.controls?.state.hideTimeControls)`
condition just in `loadDashboad` that is the place where it really matters

Your concern about the time range.. this condition is fine, since the backend fills the DTO with the proper values when the time range is hidden in the pubdash config

Reply:
@juanicabanas I think you are right, we could add this condition there, it feel like it belongs there a bit more. I can't think of any issues with moving this to state manager, @torkelo do you?

Reply:
@dprokop yea, makes sense. I don't think that condition is best there, maybe even move the isEmbedded check to state manager.